js/sound-player.js

```diff
@@ -21,23 +27,36 @@ function handleSoundItemsClick(event) {
 
 soundItems.forEach(item => item.addEventListener('click', handleSoundItemsClick));
 
-function handlePlayButtonClick(event) {
+function handleVolumeUpdate(event) {
+  const slider = event.currentTarget;
+  const volume = slider.value * 0.01;
+  const audioElement = slider.closest('.sounds__item');
+  const audioItem = audioElement.dataset.description;
+
+  const audio = document.querySelector(`.audio__${audioItem}`);
+
+  audio.volume = volume;
+}
+
+volumeSliders.forEach(item => item.addEventListener('input', handleVolumeUpdate));
```

Comment:
when we get the full functionality in place, we should look at how we're organizing the code. Nothing inherently wrong here, just a good thing to keep in the back of our minds: 'will this be easy for me to understand in a week, a month, a year?'